client/generator-youtube-player/js/main.js

```diff
@@ -64,7 +64,7 @@ var YTG = (function(){
 			}
 
 			nR.timeout = 8000;
-			nR.open("GET", window.location.origin);
+			nR.open("GET", window.location.origin + "/viewer");
```

Comment:
If the viewer was opened in a private tab, when it hit window.location.origin (our server), it would get a 302 redirect because of S30, pointing it to /viewer was a quick and (seemingly) effective fix.